about-me.html

```diff
@@ -5,15 +5,56 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>About Dirt</title>
+    <link rel="stylesheet" type="text/css" href="src/styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Amatic+SC|Cutive+Mono" rel="stylesheet">
 </head>
 <body>
+    <div>
```

Comment:
This looks really good! In the future take time to consider what ```<elements></elements>``` you are using and what is semantic. Also try to use whitespace to your advantage but keep it consistent throughout your file structure!